src/docs.scss

```diff
@@ -12,4 +12,12 @@
 @import "./core/index.scss";
 @import "./product/index.scss";
 @import "./marketing/index.scss";
-@include primer-colors-dark('[data-color-mode="dark"]');
+@import "./support/mixins/color-modes.scss";
+
+@include color-mode(light) {
+  @include primer-colors-light;
+}
+
+@include color-mode(dark) {
+  @include primer-colors-dark;
+}
```

Comment:
/cc @jonrohan @colinkeany @simurai — added this mixin so we have a system-set way to define things for various color modes. This ensures that the selectors are created correctly, and also makes mode-specific overrides easily searchable.

This could also be the recommended last-ditch solution for hacks/overrides in dotcom:

```scss
// app/assets/stylesheets/hacks/hx_some_feature.scss
@include color-mode(light) {
  --hx-some-feature-var: var(--color-scale-blue-4);
}
@include color-mode(dark) {
  --hx-some-feature-var: (var--color-scale-gray-2);
}
```